bundle/language_en.properties

```diff
@@ -1510,7 +1510,7 @@ angal.userbrowser.resetpassword.btn.key
 angal.userbrowser.resetpassword.title                                                                  = Reset Password
 angal.userbrowser.retype.password.label                                                                = Retype Password:
 angal.userbrowser.selectgroup.label                                                                    = Select group:
-angal.userbrowser.step1.pleaseinsertanew.password.fmt.msg                                              = Please insert a new password (minimum of {0} characters).
+angal.userbrowser.step1.pleaseinsertanew.password.label                                                = Please insert a new password (minimum of 6 characters).
```

Comment:
Your changes should be based on the latest develop branch as you are undoing changes I have made recently.  Thanks
Besides there are just too many files to review just your changes.